knowledge-base/text-cut-or-hidden-in-page-header.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Textbox Content Disappears or Gets Cut Off Inside Of a Page-Header
+description:  The page sections' height cannot vary. If a textbox expands it's size beyond the section area due to large content it will be either rendered cut off or will disappear completely
+type: troubleshooting
+page_title: Textbox element not showing, or part of text hidden in a page section
+slug: text-cut-or-hidden-in-page-header
+position:
+tags: textbox, hidden, pageheader, page, header, expand, big text, values
+ticketid: 1495964
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+        <tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>

```

Comment:
You may include a version here (the last one) as we discussed that soon the Page sections will be allowed to grow/shrink